src/pp-annotator/viewer/ViewerContentItem.tsx

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 import AnnotationViewModel from '../annotation/AnnotationViewModel';
+import {AnnotationPriorities, annotationPrioritiesLabels} from '../consts';
+import {Popup, Modal, Button} from 'semantic-ui-react';
+import moment from 'moment';
+import 'moment/locale/pl';
```

Comment:
I guess it's better to put this in the `vendor.ts` file

Reply:
agreed